content/soarian/dstu2/general-clinical/allergy-intolerance.md

```diff
@@ -15,6 +15,8 @@ Soarian's list supports medication, food, environmental, and other allergen cate
 
 ## Terminology Bindings
 
+Medication allergens are coded in RxNorm. Negations are indicated by SNOMED codes. All other allergens (e.g., environmental substances) are returned as text only.
```

Comment:
Don't need this. The terminology table already had this. My confusion is what are non-med allergens coded as?